### ISSUE TITLE
v0.51.0-beta error message on startup "Unhandled error when attempting to analyse the next card in the queue"

### ISSUE THREAD

**Issue by notrom** [metabase/metabase#48656]
### Describe the bug

Error in logged on startup. Fresh Metabase instance, no data sources connected, running straight from jar file.

```
2024-10-11 09:16:10,415 ::::: [main] INFO  metabase.core - Metabase Initialization COMPLETE in 25.9 s
2024-10-11 09:16:11,716 ::::: [MetabaseScheduler_Worker-1] INFO  metabase.task.sweep-query-analysis - Recalculating potentially stale analysis
2024-10-11 09:16:11,718 ::::: [MetabaseScheduler_Worker-2] ERROR metabase.task.analyze-queries - Unhandled error when attempting to analyse the next card in the queue
clojure.lang.ExceptionInfo: You can't operate on a closed ResultSet!!!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118) ~[metabase.jar:?]
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:77) ~[metabase.jar:?]
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getObject(NewProxyResultSet.java:171) ~[metabase.jar:?]
	at toucan2.jdbc.read$read_column_thunk_primary_method_default$default_read_column_thunk__26091.invoke(read.clj:73) ~[metabase.jar:?]
	at toucan2.jdbc.read$read_column_thunk_after_method_default$fn__26103.invoke(read.clj:79) ~[metabase.jar:?]
	at toucan2.jdbc.read$make_column_thunk$column_thunk__26136.invoke(read.clj:136) ~[metabase.jar:?]
	at toucan2.jdbc.read$make_cached_row_num__GT_i__GT_thunk$row_num__GT_i__GT_thunk_STAR___26145$i__GT_thunk_STAR___26146$cached_column_thunk__26147.invoke(read.clj:180) ~[metabase.jar:?]
	at toucan2.jdbc.row$fetch_column_with_name.invokeStatic(row.clj:36) ~[metabase.jar:?]
	at toucan2.jdbc.row$fetch_column_with_name.invoke(row.clj:26) ~[metabase.jar:?]
	at toucan2.jdbc.row.TransientRow.valAt(row.clj:211) ~[metabase.jar:?]
	at toucan2.jdbc.row.TransientRow.valAt(row.clj:192) ~[metabase.jar:?]
	at clojure.lang.KeywordLookupSite$1.get(KeywordLookupSite.java:45) ~[metabase.jar:?]
	at metabase.util$id.invokeStatic(util.cljc:452) ~[metabase.jar:?]
	at metabase.util$id.invoke(util.cljc:443) ~[metabase.jar:?]
	at metabase.util$the_id.invokeStatic(util.cljc:462) ~[metabase.jar:?]
	at metabase.util$the_id.invoke(util.cljc:455) ~[metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_STAR_.invokeStatic(analyze_queries.clj:44) [metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_STAR_.invoke(analyze_queries.clj:40) [metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_BANG_.invokeStatic(analyze_queries.clj:72) [metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_BANG_.invoke(analyze_queries.clj:68) [metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_BANG_.invokeStatic(analyze_queries.clj:70) [metabase.jar:?]
	at metabase.task.analyze_queries$analyzer_loop_BANG_.invoke(analyze_queries.clj:68) [metabase.jar:?]
	at metabase.task.analyze_queries.QueryAnalyzer.execute(analyze_queries.clj:81) [metabase.jar:?]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [metabase.jar:?]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [metabase.jar:?]
Caused by: java.sql.SQLException: You can't operate on a closed ResultSet!!!
	... 25 more
Caused by: java.lang.NullPointerException: Cannot invoke "java.sql.ResultSet.getObject(int)" because "this.inner" is null
	at com.mchange.v2.c3p0.impl.NewProxyResultSet.getObject(NewProxyResultSet.java:165) ~[metabase.jar:?]
	... 22 more
```

I've ignored the error and can continue to use the system with no other noticeable issues.

This message is generated each time I start that Metabase instance.

### To Reproduce

1. With v0.51.0-beta jar file
2. `java -jar -jar metabase.jar`
3. Error is displayed
4. Error generates each time I run that instance

### Expected behavior

No errors.

### Logs

As listed.

### Information about your Metabase installation

{
  "browser-info": {
    "language": "en-NZ",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "metabase-info": {
    "databases": [
      "h2"
    ],
    "run-mode": "prod",
    "plan-alias": "",
    "version": {
      "date": "2024-10-08",
      "tag": "v0.51.0-beta",
      "hash": "f69afc2"
    },
    "settings": {
      "report-timezone": null
    },
    "hosting-env": "unknown",
    "application-database": "h2",
    "application-database-details": {
      "database": {
        "name": "H2",
        "version": "2.1.214 (2022-06-13)"
      },
      "jdbc-driver": {
        "name": "H2 JDBC Driver",
        "version": "2.1.214 (2022-06-13)"
      }
    }
  },
  "system-info": {
    "file.encoding": "Cp1252",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "17.0.2+8",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "17.0.2",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "17.0.2+8",
    "os.name": "Windows 10",
    "os.version": "10.0",
    "user.language": "en",
    "user.timezone": "Pacific/Auckland"
  }
}

### Severity

Minor

**Comment by perivamsi** [comment]
I think this is a known issue

@metabase/core-backend-components

**Comment by qnkhuat** [comment]
yea, dup of https://github.com/metabase/metabase/issues/48550#issuecomment-2408071420

**Comment by crisptrutski** [comment]
This should be fixed in the next beta (`v1.50.0.1-beta`)
